### ISSUE TITLE
Ember.set on parentView of accordion error

### ISSUE THREAD

**Issue by mmahalwy** [kaliber5/ember-bootstrap#136]
```
ember.debug.js:19088 Uncaught Error: Assertion Failed: You must use Ember.set() to set the `parentView` property (of <hubble@component:bs-accordion-item::ember1285>) to `null`.
``` 

when navigating to page and removing the accordion.

**Comment by simonihmig** [comment]
What ember-bootstrap and ember.js versions are you using. There was this error in Ember 2.8, that was fixed in ember-bootstrap 0.10.0!

**Comment by simonihmig** [comment]
Closing this as this should be fixed already. Feel free to reopen when this problems is not solved for you, and please provide the details then!
